### DISCUSSION TITLE
Sort referenced Documents

### DISCUSSION THREAD

**Discussion by Amtech281** [firecmsco/firecms#181] [Q&A] [answered]
Hi
We have a collection "city" and a collection "region". In the document "city" we assign the "region".
Currently the list of regions is not sorted, is it possible to sort the list of region, when the user chooses the region in the city editform?

And is it possible to sort the Filter-List "Region" in the city collection?

Both should be sorted by name, otherwise it is hard to choose among 200+ regions

thanks

**Comment by fgatti675** [comment, marked as answer]
Hi Andre,
If you mean sorting in the popup dialog you get when selecting a reference,  you can set a default sort, so whenever that view is opened it is sorted by the specified property. It is called `initialSort` in the entity collection config.
For this use case, since the regions are probably static data, I'd consider simply using a string enumeration.
Or if you need to use a collection, you can enable text search to make it easier to retrieve.
Best,
Francesco

**Reply by Amtech281** [reply]
Great, thanks!
